stable/sentry/templates/secrets.yaml

```diff
@@ -22,9 +22,9 @@ data:
   {{ else }}
   user-password: {{ randAlphaNum 16 | b64enc | quote }}
   {{ end }}
-  {{ if or (not .Values.postgresql.enabled) (.Values.postgresql.password) }}
+  {{ if and (.Values.postgresql.existingSecret) (or (not .Values.postgresql.enabled) (.Values.postgresql.password)) }}
```

Comment:
Why need it here? It looks wrong for me. Originally password will save to this secret only if `postgresql.enabled` is set to `false` and have password settings in `postgresql.password`, which allow using this kind of config:

```yaml
postgresql:
  # internal PG is disabled, not need to deploy postgresql to kubernetes
  enabled: false
  persistence:
    enabled: false
  # use RDS settings here
  postgresqlHost: 99.111.222.333
  postgresqlDatabase: sentry
  postgresqlUsername: sentry
  postgresqlPassword: "my_db_password_here"
  postgresqlPort: 5432
```

Which allow Sentry to connect to external PostgreSQL instance.

Reply:
Nice catch. I missed this.